homeassistant/components/mutesync/strings.json

```
@@ -0,0 +1,22 @@
+{
+  "title": "m\u00fctesync",
+  "config": {
+    "step": {
+      "user": {
+        "data": {
+          "host": "[%key:common::config_flow::data::host%]",
+          "username": "[%key:common::config_flow::data::username%]",
+          "password": "[%key:common::config_flow::data::password%]"
```

Remove the ones you don't need
```suggestion
          "host": "[%key:common::config_flow::data::host%]"
```